src/EventLoop.hs

```diff
@@ -140,7 +141,7 @@ runLogicEventLoop
   -> (ProjectState -> m ())
   -> ProjectState
   -> m ProjectState
-runLogicEventLoop triggerConfig projectConfig runTime runGit runGithub getNextEvent publish initialState =
+runLogicEventLoop triggerConfig projectConfig mergeWindowExemption runTime runGit runGithub getNextEvent publish initialState =
```

Comment:
Nit: This line and the line that sets `state1` are now a bit longer than 100 characters